spec/actions_specs/git_commit_spec.rb

```diff
@@ -3,10 +3,10 @@
     describe "git_commit" do
       it "generates the correct git command" do
         result = Fastlane::FastFile.new.parse("lane :test do
-          git_commit(path: './README.md', message: 'message')
+          git_commit(path: './fastlane/README.md', message: 'message')
```

Comment:
When I was running this test, it was searching for README.md from a different location. And it couldn't find it. I checked that location and saw that README.md was located ./fastlane/README.md relative to the location of the running test.